api_spec.yaml

```diff
@@ -304,59 +304,90 @@ paths:
                       type: string
                     clientId:
```

Comment:
remove the idea of returning EXPIRED_AUTH_CODE or EXPIRED_* as status code

Reply:
changes made.